SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialPendingNetGame.h

```diff
@@ -19,4 +19,9 @@ class USpatialPendingNetGame : public UPendingNetGame
 
 	virtual void SendJoin() override;
 
+#if ENGINE_MINOR_VERSION >= 25
+	//Made virtual in PendingNetGame.h, make it do nothing, as it would usually send a packet
```

Comment:
// Made virtual in PendingNetGame.h, make it do nothing, as it would usually send a packet